website_and_docs/content/ecosystem/_index.html

```diff
@@ -458,3 +458,33 @@ <h2 class="card-title" id="frameworks">Frameworks</h2>
     </table>
   </div>
 </div>
+
+<div class="row justify-content-left pt-5 pl-5">
+  <h2 id="tools">Tools</h2>
+</div>
+<div class="row justify-content-center p-5">
+  <div class="table-responsive">
+    <table class="table">
+      <thead>
+      <tr>
+        <th scope="col">Name</th>
+        <th scope="col">Language</th>
+        <th scope="col">Author</th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr>
+        <th scope="row">
+          <p>
+            <a href="https://testcontainers.com/">
+              Testcontainers
+            </a>
+          </p>
+        </th>
+        <td>C#, Java, Python</td>
```

Comment:
Once the PR in Testcontainers for .NET is merged, .NET can be added instead of C#

```suggestion
        <td>Java, Python</td>
```

Reply:
It will be merged soon 
I Keep it open

Reply:
I think it will be faster to merge this after applying the suggestions instead of waiting until it's not only merge but release it in testcontainers-dotnet. It will also easy to add `.Net` later in this section.